tests/test_config.py

```diff
@@ -53,8 +53,30 @@ class TestConfiguration(object):
             dedent("""\
             registry:
               url: https://container-registry.example.com/v2
-              auth:
-                cfg_path: /var/run/secrets/atomic-reactor/v2-registry-dockercfg
+            registries_cfg_path: /var/run/secrets/atomic-reactor/v2-registry-dockercfg
+            """),
+            None,
+        ),
+        (
+            dedent("""\
+            registry:
+              url: https://container-registry.example.com/v2
+            """),
+            None,
+        ),
+        (
+            dedent("""\
+            registry:
+              url: https://container-registry.example.com/v2
+            registries_cfg_path: /var/run/secrets/atomic-reactor/v2-registry-dockercfg
+            """),
+            None,
+        ),
+        (
+            dedent("""\
+            registry:
+              url: https://container-registry.example.com/v2
+            registries_cfg_path: /var/run/secrets/atomic-reactor/v2-registry-dockercfg
```

Comment:
Aren't both these cases duplicates now?

Reply:
I meant both of these should be removed, the same test case already exists a few lines above this preview